src/index.html

```diff
@@ -1,22 +1,109 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="stylesheet"
+      href="Style.css"
+    />
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <title>Document</title>
+    <link rel="styleet" />
```

Comment:
There is a typo in the `rel` attribute value. It should be `stylesheet` instead of `styleet`.